src/commands/gate/evaluate.ts

```diff
@@ -58,15 +65,20 @@ export class GateEvaluateCommand extends Command {
     const gitSpanTags = await getGitMetadata()
     const userGitSpanTags = getUserGitSpanTags()
 
+    const envVarTags = this.config.envVarTags ? parseTags(this.config.envVarTags.split(',')) : {}
+    const cliTags = this.tags ? parseTags(this.tags) : {}
+
     return {
       ...gitSpanTags,
       ...ciSpanTags,
       ...userGitSpanTags,
+      ...cliTags,
+      ...envVarTags,
```

Comment:
What happens if the cli tags collide with the env tags? Let's say a user specifies `git.commit.sha` or `git.repository_url` in the cli. Which ones take precedence?

Reply:
@juan-fernandez Correct me if I'm wrong, but the latest tags take precedence here. This gives the user the ability to overwrite the default tags that we retrieve if they want, similarly to other commands (junitxml, sarif)

Reply:
👋🏼 I think I've found a way to do it in the backend without changing all the ingestion pipeline. I'm trying this before adding the `ddTags` key in the client. But as Juan commented, it depends on the specific ingestion pipeline logic for the gate command.

Reply:
Mmm I'm not sure I'm getting what the issue was with the tags for the `sarif` command. I guess in the junitxml command they are treated in the same way and nothing was spotted so I'm trying to understand whether this can be an issue for us as well or not

Reply:
We chatted in Slack. The issue was that the tags weren't deserialized correctly in the `sarif-processor` service because the corresponding struct only allowed specific fields. In the `quality-gates` service we are using a general `map[string]string` (similarly to `junitxml-processor`) so it shouldn't be a problem for us 👍

Reply:
Chatted offline, it won't be an issue for us